src/ansys/simai/core/data/types.py

```diff
@@ -262,3 +263,10 @@ def get_object_from_identifiable(
         return get_object_from_identifiable(default, directory)
     else:
         raise InvalidArguments(f"Argument {identifiable} is neither a data model nor an ID string.")
+
+
+class SubsetEnum(str, Enum):
```

Comment:
If it works, disregard, if you're having issues, know that it's because [the behavior changed since python3.11](https://github.com/python/cpython/issues/100458) and that the [workaround](https://github.com/equinor/webviz-ert/pull/449/files) looks like that.

Since it's starting to be complicated, keep in mind we can just create a type `SubSetEnum = Literal["Ignored", "Training", "Validation", "Test"]` instead